app/src/shims.d.ts

```diff
@@ -15,5 +15,6 @@ interface ImportMeta {
   env: {
     VITE_BLOCKNATIVE_API_KEY: string;
     VITE_INFURA_ID: string;
+    VITE_FLEEK_API_KEY: string;
```

Comment:
Worth renaming it to VITE_FLEEK_STORAGE_API_KEY ? (cause there is hosting key as well and it might confuse folks)